app/src/main/kotlin/com/wire/android/ui/WireActivity.kt

```diff
@@ -94,6 +100,8 @@ class WireActivity : AppCompatActivity() {
 
     val viewModel: WireActivityViewModel by viewModels()
 
+    val featureFlagNotificationViewModel: FeatureFlagNotificationViewModel by viewModels()
+
```

Comment:
I am kind of not the biggest fan of multiple viewmodes per activity, what bother me here is that it is clear what featureFlagNotificationViewModel is doing, but what is viewModel doing ? what is its responsibility ? by introducing some finer responsibilites of the smaller view model we kind of lose clearity what is the purpose of the more "global" viewmodel that is WireActivityViewModel, if we want to make it more fine grained, lets also make WireActivityViewModel more fine-grained as featureFlagNotiifcationViewMOdel otherwise we will quickly lose the track of what, each viewmodel should be doing.

Reply:
What do you mean by:
> make WireActivityViewModel more fine-grained as featureFlagNotiifcationViewMOdel

Reply:
I am fine leaving it like this, let me know if you are curious what I meant here, we can do it 1 on 1